### PR TITLE
khronos: init at 1.0.5

### PR DESCRIPTION
**Pull request by xiorcale**
###### Motivation for this change

Add more curated app from Elementary App Center to NixOS.

I started this derivation a while back and I just saw that a Nix file (https://github.com/lainsce/khronos/commit/5a7b28814e39a4f29621f0f6c09e3fc4d1e3735d) has been added to the app repository, which reminded me that mine still wasn't a PR :smile: 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by eonpatapon — APPROVED** [review]
Build & runs fine

**Review by doronbehar — APPROVED** [review]

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -19922,6 +19922,8 @@ in
     ffmpeg = ffmpeg_2;
   };
 
+  khronos = callPackage ../applications/misc/khronos { };
```
Comment (worldofpeace) [inline pkgs/top-level/all-packages.nix]: how about applications/office ?

Comment (kjuvi) [reply pkgs/top-level/all-packages.nix]: Sounds good

**Review by doronbehar — APPROVED** [review]

**Comment by worldofpeace** [comment]
Thank you @kjuvi, I've merged your PR :sparkles:

**Pull request merged**